pysaliency/filter_datasets.py

```diff
@@ -232,3 +232,50 @@ def filter_stimuli_by_size(stimuli, fixations, size=None, sizes=None):
     indices = [i for i in range(len(stimuli)) if stimuli.sizes[i] in sizes]
 
     return create_subset(stimuli, fixations, indices)
+
+
+def filter_scanpaths_by_attribute(scanpaths:  FixationTrains, attribute_name, attribute_value, invert_match=False):
+    """Filter Scanpaths by values of scanpath attribute (fixation_trains.scanpath_attributes)"""
+    
+    mask = np.array([element == attribute_value for element in getattr(scanpaths, attribute_name)])
```

Comment:
shouldn't this also work and be much more concise?
```suggestion
    mask = np.asarray(getattr(scanpaths, attribute_name)) == attribute_value
```

Reply:
yes it does work, I'll change it :)